`make deb-utils` fails if pam_zfs_key was not built

### System information
<!--  add version after "|" character -->
Type | Version/Name
 --- | ---
Distribution Name	| Ubuntu
Distribution Version	| 20.04
Kernel Version	| N/A
Architecture	| N/A
OpenZFS Version	| 0dd34a19
<!--
Command to find OpenZFS version:
zfs version

Commands to find kernel version:
uname -r		# Linux
freebsd-version -r	# FreeBSD
-->

### Describe the problem you're observing
As the title says.
```
zfs_2.1.99-1081_amd64.deb generated
libnvpair3_2.1.99-1081_amd64.deb generated
libuutil3_2.1.99-1081_amd64.deb generated
libzfs5_2.1.99-1081_amd64.deb generated
libzpool5_2.1.99-1081_amd64.deb generated
libzfs5-devel_2.1.99-1081_amd64.deb generated
zfs-test_2.1.99-1081_amd64.deb generated
zfs-dracut_2.1.99-1081_amd64.deb generated
zfs-initramfs_2.1.99-1081_amd64.deb generated
python3-pyzfs_2.1.99-1081_amd64.deb generated
File "pam_zfs_key-2.1.99-1081_g0dd34a195.x86_64.rpm" not found.
make: *** [Makefile:1748: deb-utils] Error 1
```

### Describe how to reproduce the problem
* Be missing the dependencies for pam_zfs_key
* Try `make deb-utils`

### Include any warning/errors/backtraces from the system logs

--- a/config/deb.am
+++ b/config/deb.am
@@ -61,8 +61,8 @@ deb-utils: deb-local rpm-utils-initramfs
 	pkg7=$${name}-test-$${version}.$${arch}.rpm; \
 	pkg8=$${name}-dracut-$${version}.noarch.rpm; \
 	pkg9=$${name}-initramfs-$${version}.$${arch}.rpm; \
-	pkg10=`ls python*-pyzfs-$${version}* | tail -1`; \
-	pkg11=pam_zfs_key-$${version}.$${arch}.rpm; \
+	pkg10=`ls python3-pyzfs-$${version}.noarch.rpm 2>/dev/null`; \
+	pkg11=`ls pam_zfs_key-$${version}.$${arch}.rpm 2>/dev/null`; \
 ## Arguments need to be passed to dh_shlibdeps. Alien provides no mechanism
 ## to do this, so we install a shim onto the path which calls the real
 ## dh_shlibdeps with the required arguments.

